lib/open_project/patches/primer_page_header_breadcrumb.rb

```diff
@@ -40,6 +40,6 @@ def with_breadcrumbs(breadcrumbs, skip_home_on_mobile: false, **)
   end
 end
 
-OpenProject::Patches.patch_gem_version "openproject-primer_view_components", "0.71.0" do
+OpenProject::Patches.patch_gem_version "openproject-primer_view_components", "0.72.0" do
   Primer::OpenProject::PageHeader.prepend OpenProject::Patches::PrimerPageHeaderBreadcrumb
 end
```

Comment:
@HDinger is this patch still necessary?

Reply:
Yes, it is (and will most likely be for a long time). This ensures, that the instance name is always prepended to the Breadcrumb.